update_pr.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+import requests
+
+def process_github_event(comment_url, issue_url, github_token, pr_title, pr_body, jira_key):
+    """
+    Add a rocket reaction to a comment and update a pull request title and body with a JIRA key.
+    """
+    headers = {
+        "Accept": "application/vnd.github+json",
+        "Authorization": f"Bearer {github_token}"
+    }
+
+    # Add a rocket reaction to the comment
+    reaction_url = f"{comment_url}/reactions"
+    reaction_payload = {"content": "rocket"}
+    reaction_response = requests.post(
+        reaction_url,
+        headers={**headers, "Accept": "application/vnd.github.squirrel-girl-preview+json"},
+        json=reaction_payload
+    )
+
+    if reaction_response.status_code == 201:
```

Comment:
[the API returns 200](https://docs.github.com/en/rest/reactions/reactions?apiVersion=2022-11-28#create-reaction-for-an-issue-comment) if the reaction already exists, might be worth accepting it as well. Or just do the usual `code >= 200 and code < 300`.